examples/example_dcrt.py

```diff
@@ -0,0 +1,72 @@
+"""
+This example compares the performance of d0crt based on
+the lasso (1) and random forest (2) implementations. The number of
+repetitions is set to 100. The metrics used are the type-I error and
+the power
+""" 
+
+import numpy as np
+from scipy.linalg import toeplitz
+from hidimstat.dcrt import dcrt_zero
+
+dcrtLasso_type1 = []
+dcrtLasso_power = []
+dcrtForest_type1 = []
+dcrtForest_power = []
+
+for sim_ind in range(100):
+    print(f"Processing: {sim_ind+1}")
+    np.random.seed(sim_ind)
+    DEBUG = False
+
+    n = 1000 if not DEBUG else 10
+    p = 10 if not DEBUG else 5
+    n_signal = 2 if not DEBUG else 1
+    snr = 4
+    rho = 0.8
+
+    # Create Correlation matrix with the toeplitz design
```

Comment:
This is not a Toeplitz matrix but a 1-block structure. To create a Toeplitz matrix, you need to specify diagonals.

Reply:
I've changed the design to target the standard design of the correlation matrix but I've missed correcting the comment